src/commands/Minion/alch.ts

```diff
@@ -82,6 +82,10 @@ export default class extends BotCommand {
 				break;
 			}
 		}
+		let str = [];
+		if (fireRuneCost > 0) {
+			str.push(`${fireRuneCost}x Fire rune, `);
+		}
```

Comment:
This is weird making an array just to put 1 thing into it, you could use string concatenation instead below:

`${fireRuneCost > 0 ? `${fireRuneCost}x Fire rune` : ''}`